### ISSUE TITLE
[Feature Request]: Autorename to edit file name 

### ISSUE THREAD

**Issue by ocean-haiyang** [Stirling-Tools/Stirling-PDF#1478]
### Feature Description

I want to remove document passwords then get rid of the file name that's added

### No Duplicate of the Feature

- [X] I have verified that there are no existing features requests similar to my request.

**Comment by Frooodle** [comment]
To be clear you mean to remove the appended "password_removed"?

**Comment by ocean-haiyang** [comment]
> To be clear you mean to remove the appended "password_removed"?
> 
> And I'm assuming you don't mean this to be functionality within our auto rename?

Yes that’s what I mean. The preferable solution would be to be able to edit the text that’s added or have nothing added. 

My use case is unlocking a bunch of site to based pdfs and I don’t want to have to rename them individually when I can just move them and they be replaced.

thanks!!!!!!
